sormas-ui/src/main/java/de/symeda/sormas/ui/utils/PhoneNumberValidator.java

```diff
@@ -28,7 +28,7 @@ public PhoneNumberValidator(String errorMessage) {
 
 	@Override
 	protected boolean isValidValue(String phoneNumber) {
-		return phoneNumber == null || phoneNumber.isEmpty() || phoneNumber.startsWith("+");
+		return (phoneNumber == null || phoneNumber.isEmpty() || phoneNumber.startsWith("+")) && !phoneNumber.matches(".*[a-zA-Z].*");
```

Comment:
Like written above, we can't go with this approach; this will force users to always use international phone numbers for persons and I don't think that's a good idea. Please:

* Revert the change to the PhoneNumberValidator and rename it to UserPhoneNumberValidator
* Create a new PhoneNumberValidator class that includes the check you've written, but does NOT check for "startsWith("+")" and use that for the person's phone number